stetho/src/main/java/com/facebook/stetho/inspector/protocol/module/Page.java

```diff
@@ -68,20 +69,55 @@ private void sendWelcomeMessage(JsonRpcPeer peer) {
   // Dog science...
   @ChromeDevtoolsMethod
   public JsonRpcResult getResourceTree(JsonRpcPeer peer, JSONObject params) {
+    // The DOMStorage module expects one key/value store per "security origin" which has a 1:1
+    // relationship with resource tree frames.
+    List<String> prefsTags = SharedPreferencesHelper.getSharedPreferenceTags(mContext);
+    Iterator<String> prefsTagsIter = prefsTags.iterator();
+
+    FrameResourceTree tree = createSimpleFrameResourceTree(
+        "1",
+        null /* parentId */,
+        "Stetho",
+        prefsTagsIter.hasNext() ? prefsTagsIter.next() : "");
```

Comment:
All frames require a security origin.  All security origins show up as subgroups in "Local Storage".  Therefore, the outer frame's origin is basically the first in the list.  If I do the appropriately logical thing for Andriod, a dummy row will appear that doesn't refer to SharedPreferences.